src/test/scala/zio/sqs/ZioSqsSpec.scala

```diff
@@ -1,39 +1,38 @@
 package zio.sqs
 
-import io.github.vigoo.zioaws
-import io.github.vigoo.zioaws.sqs.Sqs
-import io.github.vigoo.zioaws.sqs.model.Message
+import zio.aws.sqs.Sqs
+import zio.aws.sqs.model.Message
 import zio._
-import zio.clock.Clock
-import zio.duration._
-import zio.random.Random
+import zio.Clock
+import zio.durationInt
+import zio.Random
 import zio.sqs.ZioSqsMockServer._
 import zio.sqs.producer.{ Producer, ProducerEvent }
 import zio.sqs.serialization.Serializer
 import zio.stream.Sink
 import zio.test.Assertion._
 import zio.test._
-import zio.test.environment.{ Live, TestClock, TestEnvironment }
+import zio.test.{ Live, TestClock, TestEnvironment }
 
 object ZioSqsSpec extends DefaultRunnableSpec {
 
   def spec: ZSpec[TestEnvironment, Any] =
     suite("ZioSqsSpec")(
-      testM("send messages") {
+      test("send messages") {
         val settings: SqsStreamSettings = SqsStreamSettings(stopWhenQueueEmpty = true)
 
         for {
-          messages <- gen.sample.map(_.value).run(Sink.head[Chunk[String]]).someOrFailException
+          messages <- gen.sample.map(_.map(_.value).get).run(Sink.head[Chunk[String]]).someOrFailException
```

Comment:
Is there a better alternative to `.get` here?

Reply:
You can do `gen.runHead.someOrFailException` directly.